src/lib/openai.js

```diff
@@ -0,0 +1,53 @@
+import { OpenAI } from 'openai';
+
+function openAiConfig() {
+  const apiKey = process.env.REACT_APP_OPENAI_API_KEY;
+  if (!apiKey) {
+    throw new Error('Missing OpenAI API key in environment variables');
+  }
+  return {
+    apiKey,
+    dangerouslyAllowBrowser: true,
+  };
+}
+
+const SYSTEM_MESSAGE = 'Create a full-body, head-to-toe cartoon-style 2D digital character with bold outlines, flat colors, and expressive features. The character should reflect all given attributes (hairstyle, skin tone, body type, height, clothing, props, emotion, and action). Use a playful, clean, and polished art style with exaggerated proportions. Center the character on a neutral background. No text, logos, or extra shapes — only the character.'
```

Comment:
the last 2 renders I generated still had some extra shapes around the character. this is where prompt instruction details start to be important. See how to update the prompt structure for full body renders and also no extra shapes besides the character. These straight forward prompts don't appear to take the instruction explicitly.

Check out prompt strategies that include CRITICAL, IMPORTANT, and in use markdown structure for instructions